test/Main.hs

```diff
@@ -126,6 +126,10 @@ roundTripTests = testGroup "Roundtrip tests"
                            , roundTrip "bool"
                                        (Encode.bool (fieldNumber 1))
                                        (one Decode.bool False `at` fieldNumber 1)
+                           , roundTrip
+                               "string"
+                               (Encode.string 1)
+                               (one Decode.string "a utf8 stringよ?よ" `at` 1)
```

Comment:
Thank you for adding a test, but what is the motivation for providing a decode-to-String feature?  Do you plan to actually serialize to and from `String` in a real program?

(Please note that proto3-suite does not make use of `String` in this way.)

Though it is true we already have a `String` encoder, I am not sure that it was ever really used, and might be dead code.  Certainly the prior absence of a decoder suggests that it is not really more than a demonstration.

If you have a use for serializing to and from the `String` type, then great.  I would just like to know that you do.